code/game/objects/prop.dm

```diff
@@ -11,6 +11,33 @@
 	w_class = SIZE_SMALL
 	garbage = TRUE
 
+/obj/item/prop/geiger_counter
```

Comment:
Feels like mhelp "what does the geiger counter do" bait, maybe have it say something about the meter being broken when you turn it on?